automl/automl-script/readme.md

```diff
@@ -42,9 +42,29 @@ The **inputs** for the script are:
   `automl-execution` should be provided for the script to work. When
   both are provided, the `automl-execution` argument is discarded and
   new resources are generated from the training data.
-* `excluded-fields`: (list) List of fields that will be excluded from
-  any dataset before any other process starts. e.g. ["bmi",
-  "age"]. Empty by default
+* `configuration-params`: (map) Execution configuration
+  parameters. They will be overwritten if automl-execution is
+  given. Overwritten if automl-execution is given. The following
```

Comment:
Duplicated "overwritten if automl-execution is given".

Reply:
Removed!